packages/backend-app-api/src/services/implementations/cache/cacheServiceFactory.ts

```diff
@@ -20,7 +20,10 @@ import {
   createServiceFactory,
 } from '@backstage/backend-plugin-api';
 
-/** @public */
+/**
```

Comment:
Should we move the deprecated files to a `deprecated` folder? Just for making it easy to cleanup later?

Reply:
Ah yeah that could really help!